core/templates/pages/exploration-editor-page/editor-tab/graph-directives/exploration-graph.component.html

```diff
@@ -43,7 +43,7 @@
     </mat-card>
   </div>
   <div class="oppia-exp-graph-modal-icon-container" (click)="openStateGraphModal()">
-    <span class="material-icons">settings_overscan</span>
+    <i class="fa fa-window-maximize"></i>
```

Comment:
Can you used https://fontawesome.com/v3/icon/fullscreen ?

Reply:
Done

Reply:
Done